src/Tizen.GenericTextClassifier/Tizen.GenericTextClassifier/GenericTextClassifier.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+
+namespace Tizen.GenericTextClassifier
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct CategoryListStruct
```

Comment:
What is the point of public struct with all internal fields?

Reply:
struct is public to provide the access of it to the application developer.

Reply:
Appdeveloper can't access all member of `CategoryListStruct`. because it all internal